src/main/java/me/tomsdevsn/hetznercloud/UrlBuilder.java

```diff
@@ -0,0 +1,47 @@
+package me.tomsdevsn.hetznercloud;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class UrlBuilder {
+
+    private String url;
+
+    private Map<String, String> queryParams = new HashMap<>();
+
+    private UrlBuilder(String url) {
+        this.url = url;
+    }
+
+    public static UrlBuilder from(String url) {
+
```

Comment:
This class has a lot of unnecessary empty newlines. Please remove them.

Reply:
should be cleaner now